README.md

```diff
@@ -1 +1,202 @@
-# terraform-prefect-ecs-worker
+# Terraform Prefect ECS Module
+
+This module deploys a Prefect worker onto ECS Fargate using
+[Terraform](https://www.terraform.io/).
+
+![image](https://github.com/PrefectHQ/prefect-recipes/assets/68969861/d148af90-58dd-4ce2-a160-e23fada6c895)
+
+## Requirements
+
+You will need an AWS user with the following permissions. This will allow
+Terraform to create and manage the resources required to create the Prefect
+worker in ECS. Some example policies that provide these permissions are
+mentioned as a starting point, but we recommend providing more restricted
+access.
+
+- `ec2:CreateVpc` (provided by `AmazonVPCFullACcess` policy)
+- `ecs:CreateCluster` (provided by `AmazonECS_FullAccess` policy)
+- `secretsmanager:CreateSecret` (provided by `SecretsManagerReadWrite` policy)
+- `iam:CreateRole` (provided by `IAMFullAcess`)
+- `logs:createLogGroup` (provided by `CloudWatchLogsFullAccess` policy)
+- `ecr:CreateRepository` (provided by `AmazonEC2ContainerRegistryFullAccess` policy)
+
+Next, you will need:
+
+- The [Terraform CLI](https://developer.hashicorp.com/terraform/cli)
+- The [Prefect CLI](https://docs.prefect.io/v3/get-started/install)
+- Your Prefect account ID
+- Your Prefect workspace ID
+- Your Prefect API key
```

Comment:
(if applicable)?

Reply:
Is this in regard to the API key? I think in this case at least, it'd be required because the endpoint is [hard-coded to prefect.cloud](https://github.com/PrefectHQ/terraform-prefect-ecs-worker/blob/7fe1b1ad94a638bd9da46fd7ed4a2cca9caba882/ecs.tf#L28). We could definitely change that, though, to support self-hosted environments.